CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
This field is redundant and violates encapsulation. It duplicates the 'Salary' property and uses a lowercase name, which breaks naming conventions. Temporary fields like this should not be committed to version control.

```suggestion
// public string salary;  // Employee compensation - stores amount
```